acceptance/localcluster/localcluster.go

```diff
@@ -332,6 +332,7 @@ func (l *Cluster) processEvent(e dockerclient.EventOrError) bool {
 	// An event on any other container is unexpected. Die.
 	select {
 	case <-l.stopper:
+	case <-monitorStopper:
```

Comment:
Why will calling Stop twice panic. `l.monitorStopper` is set to `nil` to prevent closing it twice. I think I'm missing something about what this change is fixing.